src/Makefile.am

```diff
@@ -624,10 +622,6 @@ test_cache_digest: test_cache_digest.o CacheDigest.o debug.o globals.o store_key
 ## If autodependency works well this is not needed anymore
 cache_cf.o: cf_parser.cci
 
-# cf_gen builds the configuration files.
-cf_gen$(EXEEXT): $(cf_gen_SOURCES) $(cf_gen_DEPENDENCIES) cf_gen_defines.cci
```

Comment:
Removal of this rule causes cross-compile to break with "cf_gen.exe: cannot execute binary file: Exec format error"

Reply:
reverted